workflows/genetic-demux/mpileup.nf

```diff
@@ -42,92 +30,74 @@ process mpileup{
     """
 }
 
+process mpileup_cellsnp{
```

Comment:
If this isn't currently being used right now can we add a comment that states that it's still being tested or not currently used in the workflow?